.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -1,7 +1,7 @@
 name: Bug Report 🐞
 description: Report a bug and help USWDS improve.
 title: "USWDS - Bug: [YOUR TITLE]"
-labels: ['Type: Bug','Status: Triage','Needs: Confirmation']
+labels: ['Type: Bug','Needs: Confirmation']
```

Comment:
Are we opting to take the GH actions route over the template label route so that this label can't be removed from the templates +  to ensure issues that use the blank template also get the tag?